ocs_ci/ocs/ui/views.py

```diff
@@ -1284,19 +1284,24 @@
         By.XPATH,
     ),
     "req_capacity_dropdown_selected": (
-        "//div[@id='breakdown-card-title']/following-sibling::*//*[@class = 'pf-c-select__toggle-text']",
+        "//div[@id='breakdown-card-title']/following-sibling::*//*[@class = 'pf-c-select__toggle-text'] | "
+        "//div[@class='pf-v5-c-select ceph-capacity-breakdown-card-header__dropdown']"
+        "//*[@class='pf-v5-c-select__toggle-text']",
         By.XPATH,
     ),
     "req_capacity_dropdown_btn_one": (
-        "//div[@class='pf-c-select ceph-capacity-breakdown-card-header__dropdown']",
+        "//div[@class='pf-c-select ceph-capacity-breakdown-card-header__dropdown'] | "
+        "//div[@class='pf-v5-c-select ceph-capacity-breakdown-card-header__dropdown']",
         By.XPATH,
     ),
     "req_capacity_dropdown_btn_two": (
-        "(//span[@class='pf-c-select__toggle-arrow'])[2]",
+        "(//span[@class='pf-c-select__toggle-arrow'])[2] | "
+        "(//span[@class='pf-v5-c-select__toggle-arrow'])[2]",
         By.XPATH,
     ),
     "req_capacity_dropdown_list_option": (
-        "//*[@class='pf-c-select__menu-item' and contains(text(), '{}')]",
+        "//*[@class='pf-c-select__menu-item' and contains(text(), '{}')] | "
+        "//*[@class='pf-v5-c-select__menu-item' and contains(text(), '{}')]",
```

Comment:
If I understand correctly the logic and `|` act as `or` here means it will try first version and if not found it will use second - so it is backward compatible with older version and new version I would prioritize the new one to be defined as first. 

As master/stable where it will get propagated will be mostly used on newer versions of OCP/ODF so it should first give a try what is more probable so:
```
"//*[@class='pf-v5-c-select__menu-item' and contains(text(), '{}')]
```
and then 
```
//*[@class='pf-c-select__menu-item' and contains(text(), '{}')]
```

so if my assumption is correct, can we adjust all of those like this?

Thanks.

Reply:
hello Petr, your understanding is correct. 
If an impact exists it counts in milliseconds. 
Additionally we don't test locators, tags or html elements themselves and the only requirement to locator is that it should be unique pointing on specific desired web-element, meaning that we don't care if the first element will work or second - it will not be counted as a bug.

More impact in this locator is from using "*", this way we are parsing all the html elements existing on dom. 
I know this was my peace of code, but if it is possible I would suggest to set a type of element we are looking - "div", "li", "input", or anything else?

Reply:
It's a button now. I'll need to check a 4.15 cluster to see how it was before the latest changes.

Reply:
I checked it in a 4.14 cluster, it was also a button.